tacs/problems/base.py

```diff
@@ -935,3 +935,46 @@ class which handles the sensitivity file writing for ESP/CAPS shape derivatives
                                 hdl.write("1\n")
                                 hdl.write(f"{struct_sens[idx].real}\n")
             return
+
+    def writeDummySensFile(self, tacsAim):
+        """
+        write a blank / dummy SensFile for caps2tacs to enable animating shape variables
+        using GIF animations (without having to run an analysis just building each mesh)
+
+        Parameters
+        ----------
+        tacsAim : tacs.caps2tacs.TacsAIM
+            class which handles the sensitivity file writing for ESP/CAPS shape derivatives
+
+        """
+
+        num_funcs = 1
+        assert tacsAim is not None
+        num_struct_dvs = len(tacsAim.thickness_variables)
+        num_nodes = self.meshLoader.bdfInfo.nnodes
+
+        if self.comm.rank == 0:
+            # open the sens file nastran_CAPS.sens and write coordinate derivatives
+            # and any other struct derivatives to it
+            with open(tacsAim.sens_file_path, "w") as hdl:
+                for func_name in ["mass"]:
```

Comment:
This looks like a lot of repeated code from the above `writeSensFile` function. Can you just re-use that function? Maybe allow the user to pass a `None` for evalFuncs in `writeSensFile` which will trigger dummy values to be written?

Reply:
Ok I removed the `writeDummySensFile` routine and changed it so that evalFuncs=None triggers the dummy sens file.